src/kanata/mod.rs

```diff
@@ -448,11 +454,18 @@ impl Kanata {
             #[cfg(feature = "tcp_server")]
             tcp_server_address: None,
             #[cfg(all(target_os = "windows", feature = "gui"))]
-            win_tray_icon: None,
+            tray_icon: None,
+            #[cfg(all(target_os = "windows", feature = "gui"))]
+            icon_match_layer_name: true,
```

Comment:
Why `true`? Should this not be `false` or parsed from the cfg?

Reply:
the overall idea here is "eager", that's why icons are searched in a bunch of places with a bunch of prefixes and extensions. So in a similar vein by default you don't need to even specify it in a config, the app will try to load an icon just by matching the layer's name

Hence `true`, I've added the option just in case someone might find it undesirable and prefers no icons